package.json

```diff
@@ -115,10 +115,11 @@
     "testem": "^3.8.0",
     "tslib": "^2.6.3",
     "typescript": "^5.5.4",
-    "v8-to-istanbul": "^9.3.0"
+    "v8-to-istanbul": "^9.3.0",
+    "vitest": "^3.0.4"
```

Comment:
there is 3.1.1 released with lots of fixes
maybe we should jump on that train as well